### PR TITLE
Update branding to 6.4.2

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Waiting until branches are open.

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
This is ready to go.

**Comment by JunTaoLuo** [comment]
@dougbu  @wtgodbe please review

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -5,13 +5,13 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>0f7f38c4fd323b26da10cce95f857f77f0f09b48</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="3.1.1-servicing.19576.9">
-      <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>f3f2dd583fffa254015fc21ff0e45784b333256d</Sha>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="3.1.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup</Uri>
```
Comment (dougbu) [inline eng/Version.Details.xml]: This locks wrong. @mmitche should the `Uri` metadata be reverted in this PR?

Comment (mmitche) [reply eng/Version.Details.xml]: Technically you could change it, but it's not necessary.  As updates happen the uri metadata will shift back to github.

File: eng/Versions.props [hunk 2]
```diff
@@ -29,8 +29,8 @@
   </PropertyGroup>
   <PropertyGroup Label="Dependencies from dotnet/core-setup">
     <MicrosoftNETCoreAppRefPackageVersion>3.1.0</MicrosoftNETCoreAppRefPackageVersion>
-    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.1.1-servicing.19576.9</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
-    <MicrosoftNETCoreAppInternalPackageVersion>3.1.1-servicing.19576.9</MicrosoftNETCoreAppInternalPackageVersion>
+    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.1.1</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>3.1.1</MicrosoftNETCoreAppInternalPackageVersion>
```
Comment (mmitche) [inline eng/Versions.props]: Should be 3.1.1-servicing.19608.4

**Pull request merged**